### ISSUE TITLE
BUG - sev-3 - All - Availability Framework: Add further context to screen reader for Update Now button

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#7281]
## What happened?

The "update now" button that can show up in the availability framework banner announces: "update now, button, double-tap to activate".

Which - to be fair - is the same as the encouraged update banner button. However that button has static content for the banner which adds context (to improve your experience, update the app). The availability framework button will have different content (written by future app teams), and we can't necessarily guarantee that it will always have enough context for screen reader users (although I imagine, if we're including the app update button, it _should_ say something like "we've released a fix, go update your app")

## Specs:

- Device:
- OS:
- User Account:
- Accessibility State:

## Acceptance Criteria

@brea11y to weigh in (I'll ping her in the availability framework channel in slack) with either suggested context, or closing the issue because it's not needed

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: High Low 
- Frequency: High Low 

- 3 - Low
- 2 - High
- 1 - Critical

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the [bug epic](https://app.zenhub.com/workspaces/va-mobile-product-view-610035bc5395bb000e62e529/issues/gh/department-of-veterans-affairs/va-mobile-app/5367)

**Comment by TKDickson** [comment]
whoops, reopened by accident, closing
